src/lib.rs

```diff
@@ -935,6 +936,14 @@ impl<'conn> Statement<'conn> {
         }
     }
 
+    fn sql(&self) -> String { // TODO Maybe SQL should by kept as an SqliteStatement field ?
```

Comment:
Couple comments:

* This may need to be `#[cfg(feature = "cache")]` flagged to avoid a dead code warning if you're compiling without the cache feature.
* Keeping the SQL as a field on `Statement` is tempting, but I'm inclined not to (and keep this as-is). SQLite is already keeping a copy around; no need to keep a second.